test-support/helpers/ember-i18n/test-helpers.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+// example usage: find(`.header:contains(${t('welcome_message')})`)
+Ember.Test.registerHelper('t', function(app, key){
+  var i18n = app.__container__.lookup('service:i18n');
+  return i18n.t(key);
+});
+
+// example usage: expectTranslation('.header', 'welcome_message');
+Ember.Test.registerHelper('expectTranslation', function(app, element, key){
+  var i18n = app.__container__.lookup('service:i18n'),
+      translation = i18n.t(key);
+
+  return ok(find(`${element}:contains(${translation})`),  `Found translation key ${key} in ${element}`);
```

Comment:
Where does `ok` come from? That looks like a QUnit global. What about for people using mocha? Sadly, [ember-test-helpers](https://github.com/switchfly/ember-test-helpers) doesn't seem to provide any cross-framework assertions.